### ISSUE TITLE
Lighthous PWA audit complain about missing icons althoug SVG is provided

### ISSUE THREAD

**Issue by Eccenux** [GoogleChrome/lighthouse#4883]
According to [W3C Manifest Standard](https://w3c.github.io/manifest/#icons-member) this is a correct manifest:
```json
{
  "icons": [
    {
      "src": "icon/lowres.webp",
      "sizes": "48x48",
      "type": "image/webp"
    },{
      "src": "icon/lowres",
      "sizes": "48x48"
    },{
      "src": "icon/hd_hi.ico",
      "sizes": "72x72 96x96 128x128 256x256"
    },{
      "src": "icon/hd_hi.svg",
      "sizes": "257x257"
    }]
}
```
But when tested with Chrome/Lighthouse it complains about missing icons for 512px.

And if you use something like below it will also complain about missing 192px icon.
```json
	"icons": [
		{
			"src": "/ico/apple-touch-icon-72-precomposed.png",
			"type": "image/png",
			"sizes": "72x72"
		},
		{
			"src": "/ico/apple-touch-icon-114-precomposed.png",
			"type": "image/png",
			"sizes": "114x114"
		},
		{
			"src": "/ico/apple-touch-icon-144-precomposed.png",
			"type": "image/png",
			"sizes": "144x144"
		},
		{
			"src": "/ico/catalogue_online.svg",
			"sizes": "145x145"
		}
	],
```
To my understanding the largest icon should simply be up-scaled... Or is it not working like that? Quote from W3C:

> The developer wants to use an SVG for greater than or equal to 257x257px. She has found that the SVG file looks too blurry at small sizes, even on high-density screens. To deal with this problem, she's included an SVG icon that is only used when the dimensions are at least 257px. Otherwise, the user agent uses the ICO file (hd_hi.ico), which includes a gamut of raster icons individually tailored for small display sizes.

**Comment by patrickhulce** [comment]
You are correct, all of these are valid manifests and will work in some capacity. The audit is intended to provide guidance on the minimal set of icons to have ideal rendering on Android homescreen and splash screen.


related discussion https://github.com/GoogleChrome/lighthouse/issues/291#issuecomment-376301460 #685
